codebase/superdarn/src.bin/tk/tool/map_addimf.1.18/map_addimf.c

```diff
@@ -312,22 +312,36 @@ int findvalue(struct swdata *ptr, double tme, float *val)
   kp  = ptr->Kp;
   cnt = ptr->cnt;
 
+  /* Fill in with default values */
   for (i=0; i<5; i++) val[i] = FILL_VALUE;
 
+  /* Increment i until end of file or until the start time */
+  /* of making the map is reached.  This is likely most */
+  /* useful for sw data that contains multiple days.  The */
+  /* default format from OMNI is a year-long file */
```

Comment:
There is no default format - OMNI data comes in both monthly and yearly file formats for 1- and 5-min resolution data.

Reply:
Great catch Evan, I guess what I meant to say was *I've* defaultly used the year-long files.  Let me know what you think of the updated comment in fe30239.